docs/en/latest/apisix-ingress-controller.md

```diff
@@ -37,6 +37,14 @@ helm repo update
 helm install apisix-ingress-controller apisix/apisix-ingress-controller --namespace ingress-apisix --create-namespace
 ```
 
+If apisix use different namespace with "ingress-apisix", please set value of "config.apisix.serviceNamespace":
```

Comment:
Could you please add another configuration?
`config.apisix.serviceName=apisix-admin`